### PR TITLE
master-lps-101051_1

### PR DESCRIPTION
**Pull request by Jeremy-Ch**
https://issues.liferay.com/browse/LPS-101051

### CODE REVIEW

**Comment by Jeremy-Ch** [comment]
ci:test:sf

File: modules/apps/depot/depot-test/src/testFunctional/macros/Depot.macro [hunk 1]
```diff
@@ -56,6 +56,32 @@ definition {
 		Alert.viewSuccessMessage();
 	}
 
+	macro gotoConfigurationSite {
```
Comment (austinchiang) [inline modules/apps/depot/depot-test/src/testFunctional/macros/Depot.macro]: Please change to connectSite

File: modules/apps/depot/depot-test/src/testFunctional/macros/Depot.macro [hunk 2]
```diff
@@ -56,6 +56,32 @@ definition {
...
+		if (isSet(connectSite)) {
```
Comment (austinchiang) [inline modules/apps/depot/depot-test/src/testFunctional/macros/Depot.macro]: You don't need this line

File: modules/apps/depot/depot-test/src/testFunctional/macros/Depot.macro [hunk 3]
```diff
@@ -56,6 +56,32 @@ definition {
...
+				SiteSelector.selectSite(site = "${siteName}");
+
+				SelectFrame(value1 = "relative=top");
+
+				Alert.viewSuccessMessage();
+			}
+		}
+	}
+
+	macro viewConnectSite {
```
Comment (austinchiang) [inline modules/apps/depot/depot-test/src/testFunctional/macros/Depot.macro]: viewConnectedSite

File: modules/apps/depot/depot-test/src/testFunctional/macros/DepotNavigator.macro [hunk 4]
```diff
@@ -25,6 +25,18 @@ definition {
 			value1 = "Edit");
 	}
 
+	macro gotoSites {
+		DepotNavigator.gotoEdit(depotName = "${depotName}");
```
Comment (austinchiang) [inline modules/apps/depot/depot-test/src/testFunctional/macros/DepotNavigator.macro]: This shouldn't be in the macro

File: modules/apps/depot/depot-test/src/testFunctional/tests/DepotIntegration.testcase [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+			DMFolder.tearDownCP();
+
+			User.logoutPG();
+
+			Site.tearDownCP();
+		}
+	}
+
+	@description = "This ensures that a content repository can be connected to the site."
```
Comment (austinchiang) [inline modules/apps/depot/depot-test/src/testFunctional/tests/DepotIntegration.testcase]: Use depot, not content repository, for consistency

**Comment by austinchiang** [comment]
Please resend after making changes

**Pull request closed without merging**